{"uid":"6a26ad57684a408a","name":"TC 03.02.02 to 03_02_10 Verify Error on Failed Login with invalid credentials","fullName":"tests.test_sign_in.TestFailedLogin#test_03_02_02_to_10_error_if_login_with_invalid_credentials","historyId":"3333058a43e38a67e1702af7603bd873","time":{"start":1728786351785,"stop":1728786353014,"duration":1229},"description":"Check error on Failed Login with invalid credentials, case-sensitive password and password with spaces","descriptionHtml":"<p>Check error on Failed Login with invalid credentials, case-sensitive password and password with spaces</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <input name=\"login[username]\" value=\"\" autocomplete=\"off\" id=\"email\" type=\"email\" class=\"input-text\" title=\"Email\" data-validate=\"{required:true, 'validate-email':true}\" aria-required=\"true\"> is not clickable at point (560, 767). Other element would receive the click: <div class=\"grippy-host\"></div>\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56388a57102a <unknown>\n#1 0x56388a2575e0 <unknown>\n#2 0x56388a2ad866 <unknown>\n#3 0x56388a2ab78d <unknown>\n#4 0x56388a2a9237 <unknown>\n#5 0x56388a2a8646 <unknown>\n#6 0x56388a29ba98 <unknown>\n#7 0x56388a2cbb22 <unknown>\n#8 0x56388a29b478 <unknown>\n#9 0x56388a2cbcee <unknown>\n#10 0x56388a2ead7d <unknown>\n#11 0x56388a2cb8c3 <unknown>\n#12 0x56388a2996b3 <unknown>\n#13 0x56388a29a68e <unknown>\n#14 0x56388a53ba2b <unknown>\n#15 0x56388a53f9b1 <unknown>\n#16 0x56388a528225 <unknown>\n#17 0x56388a540532 <unknown>\n#18 0x56388a50d38f <unknown>\n#19 0x56388a55ff28 <unknown>\n#20 0x56388a5600f3 <unknown>\n#21 0x56388a56fe7c <unknown>\n#22 0x7fa5c8694ac3 <unknown>","statusTrace":"self = <test_sign_in.TestFailedLogin object at 0x7fdac8656c90>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"25e7f7252e1538dec46bb49e7be9718c\")>\nsign_in_page = <pages.sign_in_page.SignInPage object at 0x7fdac9dd3cd0>\nemail = 'teston120@mailinator.com', password = ' Test#12p@ssword'\n\n    @allure.title('TC 03.02.02 to 03_02_10 Verify Error on Failed Login with invalid credentials')\n    @pytest.mark.parametrize('email, password', [LOGIN[i] for i in [1, 2, 6, 8, 9]])\n    def test_03_02_02_to_10_error_if_login_with_invalid_credentials(self, driver, sign_in_page, email, password):\n        \"\"\"Check error on Failed Login with invalid credentials, case-sensitive password and password with spaces\"\"\"\n>       sign_in_page.fill_in_email_field(email)\n\ntests/test_sign_in.py:271: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/sign_in_page.py:54: in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\npages/base_page.py:319: in fill_in_field\n    input_field.click()\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:93: in click\n    self._execute(Command.CLICK_ELEMENT)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:394: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fdac9cdb510>\nresponse = {'status': 400, 'value': '{\"value\":{\"error\":\"element click intercepted\",\"message\":\"element click intercepted: Element ...\\\\n#20 0x56388a5600f3 \\\\u003Cunknown>\\\\n#21 0x56388a56fe7c \\\\u003Cunknown>\\\\n#22 0x7fa5c8694ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <input name=\"login[username]\" value=\"\" autocomplete=\"off\" id=\"email\" type=\"email\" class=\"input-text\" title=\"Email\" data-validate=\"{required:true, 'validate-email':true}\" aria-required=\"true\"> is not clickable at point (560, 767). Other element would receive the click: <div class=\"grippy-host\"></div>\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x56388a57102a <unknown>\nE       #1 0x56388a2575e0 <unknown>\nE       #2 0x56388a2ad866 <unknown>\nE       #3 0x56388a2ab78d <unknown>\nE       #4 0x56388a2a9237 <unknown>\nE       #5 0x56388a2a8646 <unknown>\nE       #6 0x56388a29ba98 <unknown>\nE       #7 0x56388a2cbb22 <unknown>\nE       #8 0x56388a29b478 <unknown>\nE       #9 0x56388a2cbcee <unknown>\nE       #10 0x56388a2ead7d <unknown>\nE       #11 0x56388a2cb8c3 <unknown>\nE       #12 0x56388a2996b3 <unknown>\nE       #13 0x56388a29a68e <unknown>\nE       #14 0x56388a53ba2b <unknown>\nE       #15 0x56388a53f9b1 <unknown>\nE       #16 0x56388a528225 <unknown>\nE       #17 0x56388a540532 <unknown>\nE       #18 0x56388a50d38f <unknown>\nE       #19 0x56388a55ff28 <unknown>\nE       #20 0x56388a5600f3 <unknown>\nE       #21 0x56388a56fe7c <unknown>\nE       #22 0x7fa5c8694ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: ElementClickInterceptedException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1728783765157,"stop":1728783765226,"duration":69},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder","time":{"start":1728783765226,"stop":1728783765226,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1728786348494,"stop":1728786349698,"duration":1204},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"sign_in_page","time":{"start":1728786349699,"stop":1728786351784,"duration":2085},"status":"passed","steps":[{"name":"Open a browser","time":{"start":1728786349699,"stop":1728786351784,"duration":2085},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Check error on Failed Login with invalid credentials, case-sensitive password and password with spaces","status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <input name=\"login[username]\" value=\"\" autocomplete=\"off\" id=\"email\" type=\"email\" class=\"input-text\" title=\"Email\" data-validate=\"{required:true, 'validate-email':true}\" aria-required=\"true\"> is not clickable at point (560, 767). Other element would receive the click: <div class=\"grippy-host\"></div>\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56388a57102a <unknown>\n#1 0x56388a2575e0 <unknown>\n#2 0x56388a2ad866 <unknown>\n#3 0x56388a2ab78d <unknown>\n#4 0x56388a2a9237 <unknown>\n#5 0x56388a2a8646 <unknown>\n#6 0x56388a29ba98 <unknown>\n#7 0x56388a2cbb22 <unknown>\n#8 0x56388a29b478 <unknown>\n#9 0x56388a2cbcee <unknown>\n#10 0x56388a2ead7d <unknown>\n#11 0x56388a2cb8c3 <unknown>\n#12 0x56388a2996b3 <unknown>\n#13 0x56388a29a68e <unknown>\n#14 0x56388a53ba2b <unknown>\n#15 0x56388a53f9b1 <unknown>\n#16 0x56388a528225 <unknown>\n#17 0x56388a540532 <unknown>\n#18 0x56388a50d38f <unknown>\n#19 0x56388a55ff28 <unknown>\n#20 0x56388a5600f3 <unknown>\n#21 0x56388a56fe7c <unknown>\n#22 0x7fa5c8694ac3 <unknown>","statusTrace":"self = <test_sign_in.TestFailedLogin object at 0x7fdac8656c90>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"25e7f7252e1538dec46bb49e7be9718c\")>\nsign_in_page = <pages.sign_in_page.SignInPage object at 0x7fdac9dd3cd0>\nemail = 'teston120@mailinator.com', password = ' Test#12p@ssword'\n\n    @allure.title('TC 03.02.02 to 03_02_10 Verify Error on Failed Login with invalid credentials')\n    @pytest.mark.parametrize('email, password', [LOGIN[i] for i in [1, 2, 6, 8, 9]])\n    def test_03_02_02_to_10_error_if_login_with_invalid_credentials(self, driver, sign_in_page, email, password):\n        \"\"\"Check error on Failed Login with invalid credentials, case-sensitive password and password with spaces\"\"\"\n>       sign_in_page.fill_in_email_field(email)\n\ntests/test_sign_in.py:271: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/sign_in_page.py:54: in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\npages/base_page.py:319: in fill_in_field\n    input_field.click()\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:93: in click\n    self._execute(Command.CLICK_ELEMENT)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:394: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fdac9cdb510>\nresponse = {'status': 400, 'value': '{\"value\":{\"error\":\"element click intercepted\",\"message\":\"element click intercepted: Element ...\\\\n#20 0x56388a5600f3 \\\\u003Cunknown>\\\\n#21 0x56388a56fe7c \\\\u003Cunknown>\\\\n#22 0x7fa5c8694ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <input name=\"login[username]\" value=\"\" autocomplete=\"off\" id=\"email\" type=\"email\" class=\"input-text\" title=\"Email\" data-validate=\"{required:true, 'validate-email':true}\" aria-required=\"true\"> is not clickable at point (560, 767). Other element would receive the click: <div class=\"grippy-host\"></div>\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x56388a57102a <unknown>\nE       #1 0x56388a2575e0 <unknown>\nE       #2 0x56388a2ad866 <unknown>\nE       #3 0x56388a2ab78d <unknown>\nE       #4 0x56388a2a9237 <unknown>\nE       #5 0x56388a2a8646 <unknown>\nE       #6 0x56388a29ba98 <unknown>\nE       #7 0x56388a2cbb22 <unknown>\nE       #8 0x56388a29b478 <unknown>\nE       #9 0x56388a2cbcee <unknown>\nE       #10 0x56388a2ead7d <unknown>\nE       #11 0x56388a2cb8c3 <unknown>\nE       #12 0x56388a2996b3 <unknown>\nE       #13 0x56388a29a68e <unknown>\nE       #14 0x56388a53ba2b <unknown>\nE       #15 0x56388a53f9b1 <unknown>\nE       #16 0x56388a528225 <unknown>\nE       #17 0x56388a540532 <unknown>\nE       #18 0x56388a50d38f <unknown>\nE       #19 0x56388a55ff28 <unknown>\nE       #20 0x56388a5600f3 <unknown>\nE       #21 0x56388a56fe7c <unknown>\nE       #22 0x7fa5c8694ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: ElementClickInterceptedException","steps":[{"name":"Fill in Email field","time":{"start":1728786351785,"stop":1728786353013,"duration":1228},"status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <input name=\"login[username]\" value=\"\" autocomplete=\"off\" id=\"email\" type=\"email\" class=\"input-text\" title=\"Email\" data-validate=\"{required:true, 'validate-email':true}\" aria-required=\"true\"> is not clickable at point (560, 767). Other element would receive the click: <div class=\"grippy-host\"></div>\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56388a57102a <unknown>\n#1 0x56388a2575e0 <unknown>\n#2 0x56388a2ad866 <unknown>\n#3 0x56388a2ab78d <unknown>\n#4 0x56388a2a9237 <unknown>\n#5 0x56388a2a8646 <unknown>\n#6 0x56388a29ba98 <unknown>\n#7 0x56388a2cbb22 <unknown>\n#8 0x56388a29b478 <unknown>\n#9 0x56388a2cbcee <unknown>\n#10 0x56388a2ead7d <unknown>\n#11 0x56388a2cb8c3 <unknown>\n#12 0x56388a2996b3 <unknown>\n#13 0x56388a29a68e <unknown>\n#14 0x56388a53ba2b <unknown>\n#15 0x56388a53f9b1 <unknown>\n#16 0x56388a528225 <unknown>\n#17 0x56388a540532 <unknown>\n#18 0x56388a50d38f <unknown>\n#19 0x56388a55ff28 <unknown>\n#20 0x56388a5600f3 <unknown>\n#21 0x56388a56fe7c <unknown>\n#22 0x7fa5c8694ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/sign_in_page.py\", line 54, in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 319, in fill_in_field\n    input_field.click()\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py\", line 93, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py\", line 394, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Check Email field is clickable","time":{"start":1728786351785,"stop":1728786351822,"duration":37},"status":"passed","steps":[{"name":"Find clickable elements","time":{"start":1728786351785,"stop":1728786351822,"duration":37},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', '#email')"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Fill in a field","time":{"start":1728786351822,"stop":1728786353013,"duration":1191},"status":"broken","statusMessage":"selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <input name=\"login[username]\" value=\"\" autocomplete=\"off\" id=\"email\" type=\"email\" class=\"input-text\" title=\"Email\" data-validate=\"{required:true, 'validate-email':true}\" aria-required=\"true\"> is not clickable at point (560, 767). Other element would receive the click: <div class=\"grippy-host\"></div>\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56388a57102a <unknown>\n#1 0x56388a2575e0 <unknown>\n#2 0x56388a2ad866 <unknown>\n#3 0x56388a2ab78d <unknown>\n#4 0x56388a2a9237 <unknown>\n#5 0x56388a2a8646 <unknown>\n#6 0x56388a29ba98 <unknown>\n#7 0x56388a2cbb22 <unknown>\n#8 0x56388a29b478 <unknown>\n#9 0x56388a2cbcee <unknown>\n#10 0x56388a2ead7d <unknown>\n#11 0x56388a2cb8c3 <unknown>\n#12 0x56388a2996b3 <unknown>\n#13 0x56388a29a68e <unknown>\n#14 0x56388a53ba2b <unknown>\n#15 0x56388a53f9b1 <unknown>\n#16 0x56388a528225 <unknown>\n#17 0x56388a540532 <unknown>\n#18 0x56388a50d38f <unknown>\n#19 0x56388a55ff28 <unknown>\n#20 0x56388a5600f3 <unknown>\n#21 0x56388a56fe7c <unknown>\n#22 0x7fa5c8694ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 319, in fill_in_field\n    input_field.click()\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py\", line 93, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py\", line 394, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Find clickable elements","time":{"start":1728786351822,"stop":1728786351843,"duration":21},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"<selenium.webdriver.remote.webelement.WebElement (session=\"25e7f7252e1538dec46bb49e7be9718c\", element=\"f.E23752F76E7CA886BA729AE20EDFB683.d.74F0A5232C5414AA7A3778A11AFD9B61.e.3\")>"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"locator","value":"<selenium.webdriver.remote.webelement.WebElement (session=\"25e7f7252e1538dec46bb49e7be9718c\", element=\"f.E23752F76E7CA886BA729AE20EDFB683.d.74F0A5232C5414AA7A3778A11AFD9B61.e.3\")>"},{"name":"value","value":"'teston120@mailinator.com'"}],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"email","value":"'teston120@mailinator.com'"}],"shouldDisplayMessage":false,"stepsCount":4,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"dbdc8ead6e9d76bc","name":"Screenshot","source":"dbdc8ead6e9d76bc.png","type":"image/png","size":300414},{"uid":"59748f66db05a035","name":"HTML source","source":"59748f66db05a035.html","type":"text/html","size":131848}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":5,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1728786353194,"stop":1728786353267,"duration":73},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder::0","time":{"start":1728786423763,"stop":1728786423763,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Failed Login Functionality"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_sign_in"},{"name":"subSuite","value":"TestFailedLogin"},{"name":"host","value":"fv-az840-331"},{"name":"thread","value":"2952-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_sign_in"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'teston120@mailinator.com'"},{"name":"password","value":"' Test#12p@ssword'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"6a26ad57684a408a.json","parameterValues":["'teston120@mailinator.com'","' Test#12p@ssword'"]}